obal/__init__.py

```diff
@@ -298,11 +310,11 @@ def generate_ansible_args(inventory_path, args, obal_arguments):
     return ansible_args
 
 
-def main(cliargs=None):  # pylint: disable=R0914
+def main(cliargs=None, layout=Layout):  # pylint: disable=R0914
     """
     Main command
     """
-    cfg_path = get_ansible_config_path()
+    cfg_path = layout.ansible_config_path
```

Comment:
I was thinking whether this should incorporate the if exists logic, but if you need to create it, this can be useful.